doc/snippets/Microsoft.Data.SqlClient/SqlColumnEncryptionCngProvider.xml

```diff
@@ -1,67 +1,90 @@
+      </summary>
+      <returns>
+        The decrypted column encryption key.
+      </returns>
+    </DecryptColumnEncryptionKey>
+    <EncryptColumnEncryptionKey>
+      <param name="masterKeyPath">
+        The master key path.
+      </param>
+      <param name="encryptionAlgorithm">
+        The encryption algorithm.
+      </param>
+      <param name="columnEncryptionKey">
+        The plaintext column encryption key.
+      </param>
+      <summary>
+        Encrypts the given plain text column encryption key using an asymmetric key specified by the key path and the specified algorithm. The key path will be in the format of [ProviderName]/KeyIdentifier and should be an asymmetric key stored in the specified CNG key store provider. The valid algorithm used to encrypt/decrypt the CEK is 'RSA_OAEP'.
+      </summary>
+      <returns>
+        The encrypted column encryption key.
+      </returns>
+    </EncryptColumnEncryptionKey>
+    <ProviderName>
+      <summary>
+        A constant string for the provider name 'MSSQL_CNG_STORE'.
+      </summary>
+    </ProviderName>
+    <SignColumnMasterKeyMetadata>
+      <param name="masterKeyPath">
+        The column master key path. The path format is specific to a key store provider.
+      </param>
+      <param name="allowEnclaveComputations">
+        <see langword="true" /> to indicate that the column master key supports enclave computations; otherwise, <see langword="false" />.
+      </param>
+      <summary>
+        Throws a <see cref="T:System.NotSupportedException" /> exception in all cases.
+      </summary>
+      <returns>
+        The signature of the column master key metadata.
+      </returns>
+      <remarks>
+        The <see cref="P:Microsoft.Data.SqlClient.SqlColumnEncryptionCngProvider.SignColumnMasterKeyMetadata" /> method must be implemented by the corresponding key store providers. <see cref="P:Microsoft.Data.SqlClient.SqlColumnEncryptionCngProvider.SignColumnMasterKeyMetadata" /> should use an asymmetric key identified by a key path and sign the masterkey metadata consisting of <c>masterKeyPath</c>, <c>allowEnclaveComputations</c>, and providerName.
+      </remarks>
+    </SignColumnMasterKeyMetadata>
+    <VerifyColumnMasterKeyMetadata>
+      <param name="masterKeyPath">
+        The complete path of an asymmetric key. The path format is specific to a key store provider.
+      </param>
+      <param name="allowEnclaveComputations">
+        A Boolean that indicates if this key can be sent to the trusted enclave.
+      </param>
+      <param name="signature">
+        The master key metadata signature.
+      </param>
+      <summary>
+        This function must be implemented by the corresponding Key Store providers. This function should use an asymmetric key identified by a key path and verify the masterkey metadata consisting of (masterKeyPath, allowEnclaveComputations, providerName).
```

Comment:
Are these parameter names suppossed to be quoted literals too? As done in doc/snippets/Microsoft.Data.SqlClient/SqlColumnEncryptionCspProvider.xml (L69)

Reply:
They should be linked as <paramref> tags. I'll fix.